site/src/util/workspace.ts

```diff
@@ -4,89 +4,12 @@ import duration from "dayjs/plugin/duration"
 import minMax from "dayjs/plugin/minMax"
 import utc from "dayjs/plugin/utc"
 import semver from "semver"
-import { WorkspaceBuildTransition } from "../api/types"
 import * as TypesGen from "../api/typesGenerated"
 
 dayjs.extend(duration)
 dayjs.extend(utc)
 dayjs.extend(minMax)
 
-// all the possible states returned by the API
```

Comment:
The backend gives us the `WorkspaceStatus` type and the actual workspace status now, so we can delete this. The display language is deleted just because it's moved into i18n.